openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -211,6 +213,30 @@ class LibraryTeamView(APIView):
     Note also the 'allow_public_' settings which can be edited by PATCHing the
     library itself (LibraryDetailsView.patch).
     """
+    @convert_exceptions
+    def post(self, request, lib_key_str):
```

Comment:
We should move the validation of having "at least one admin" from edx/frontend-app-library-authoring#9 to the backend here.